crawl-ref/source/mon-gear.cc

```diff
@@ -511,6 +511,10 @@ int make_mons_weapon(monster_type type, int level, bool melee_only)
             { { SPWPN_CHAOS, 1 } } } },
         { MONS_JOSEPH, { { { WPN_QUARTERSTAFF, 1 } } } },
         { MONS_SPRIGGAN_DRUID, { { { WPN_QUARTERSTAFF, 1 } } } },
+        { MONS_JOROGUMO,
+            { { { WPN_QUARTERSTAFF,        20 },
+                { WPN_LAJATANG,            1 },
+        }, {}, {}, 1 } },
```

Comment:
One of the 'things' in Spider is that it has no item-using monsters, aside from visiting uniques. It seems easy enough to keep this intact with Jorogumos - since they're spider spirits merely pretending to be human, it makes some sense that they wouldn't be weapon using. (Likewise, they probably shouldn't have robes or `MONUSE_WEAPONS_ARMOUR`.)

If this is important to the design, please push back and let me know.